### PR TITLE
Added Logo

### PR DESCRIPTION
**Pull request by Vrushti24**
#### Issue Number
ISSUE #

#### Checklist

- [x] My code follows the code style of this project.
- [x] I have performed a self-review of my own code.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by chandrikadeb7** [comment]
@Vrushti24 Include a smaller size above gif file in Readme also.

**Comment by Vrushti24** [comment]
Yeah sure

**Comment by Vrushti24** [comment]
@chandrikadeb7 done u can check

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-![Live Demo](https://github.com/chandrikadeb7/Face-Mask-Detection/blob/master/Readme_images/Demo.gif)
+![Live Demo](https://github.com/Vrushti24/Face-Mask-Detection/blob/logo/Logo/facemaskdetection.ai%20%40%2051.06%25%20(CMYK_GPU%20Preview)%20%2018-02-2021%2018_33_18%20(2).png)
```
Comment (chandrikadeb7) [inline README.md:17]: Keep it below the heading 'Face Mask Detection'. Reduce the size of the logo. Keep it in 1:1 ratio.

**Comment by Vrushti24** [comment]
Okay will do that but actually I am out of town so ..

**Comment by Vrushti24** [comment]
Hey I m still out of town I have messaged u on discord about that is there
any other way to change the size because I don't have my laptop with me so
...

**Pull request merged**